src/featured-content.scss

```diff
@@ -30,10 +31,13 @@ a.cardClick:hover {
   text-decoration: none;
 }
 
-h3.feat-content-title {
+.feat-content-title {
   color: #487980;
   margin-top: 1rem;
 }
+.feat-content-card:hover {
+  box-shadow: 0 0.4rem 0.5rem 0 rgba(0, 0, 0, 0.5);
```

Comment:
Can you explain the rationale behind adding box-shadow (aka increasing elevation) when hovering?

Reply:
Just for pretty-ing, the box gets highlighted when you however. 
![hover-feat-content](https://user-images.githubusercontent.com/23464754/72848094-b930e780-3c58-11ea-908b-3cb2d3e18128.gif)